### PR TITLE
Adapted notebooks to run with public thalamus circuit in Zenodo

### PR DESCRIPTION
**Pull request by elisabettai**
Adapted notebooks 1-4 to run with thalamus circuit, as discussed with @joni-herttuainen and @mgeplf. 
We can do the same for simulation when it becomes available.

### CODE REVIEW

**Comment by mgeplf** [comment]
Thanks, we'll be looking at this shortly.

File: doc/source/notebooks/02_node_populations.ipynb [hunk 1]
```diff
@@ -14,7 +14,7 @@
    "metadata": {},
    "source": [
     "## Preamble\n",
-    "The code in this section is identical to the code in sections \"Introduction\" and \"Loading\" from the previous tutorial."
+    "The code in this section is identical to the code in sections \"Introduction\" and \"Loading\" from the previous tutorial. It assumumes that you have already download the circuit. It assumumes that you have already downloaded the circuit. If not, take a look to the notebook **01_circuits** (Downloading a circuit)."
```
Comment (joni-herttuainen) [inline doc/source/notebooks/02_node_populations.ipynb:17]: Small typo and a duplicate sentence:
```suggestion
    "The code in this section is identical to the code in sections \"Introduction\" and \"Loading\" from the previous tutorial. It assumes that you have already downloaded the circuit. If not, take a look to the notebook **01_circuits** (Downloading a circuit)."
```

File: doc/source/notebooks/02_node_populations.ipynb [hunk 2]
```diff
@@ -54,7 +56,7 @@
     }
    ],
    "source": [
-    "circuit.nodes"
+    "circuit.nodes.population_names"
    ]
   },
   {
```
Comment (joni-herttuainen) [inline doc/source/notebooks/02_node_populations.ipynb:62]: L66: 
```
In this example we can see there is only one node population, which is named `All`. We can access the node population using the name as the key:
```
Should be something like:
```
In this example, we can see there are several node populations. We can access a node population using its name as the key:
```

**Review by joni-herttuainen — CHANGES_REQUESTED** [review]
Only a few minor things to update. Good work!

File: doc/source/notebooks/02_node_populations.ipynb (hunk 2 above)
Comment (elisabettai) [reply doc/source/notebooks/02_node_populations.ipynb:62]: Thanks for spotting this, I've updated the text.

**Review by joni-herttuainen — COMMENTED** [review]
Looks good to me. Did you accidentally close this Pull Request?

**Comment by elisabettai** [comment]
Yes, sorry. I've re-opened it.

**Comment by joni-herttuainen** [comment]
No worries, happens to everybody 😄

**Review by joni-herttuainen — APPROVED** [review]

**Pull request merged**